src/openziti/zitisock.py

```diff
@@ -106,7 +105,9 @@ def create_ziti_connection(address, **_):
     sock.connect(address)
     return sock
 
+
 def ziti_getaddrinfo(host, port, family=0, type=0, proto=0, flags=0):
+    # pylint: disable=redefined-builtin
```

Comment:
why is this needed?

Reply:
`type` is a builtin: https://docs.python.org/3/library/functions.html#type

In this case, I assumed this came from the conventions used by the `socket` module, so I silenced the message.